packages/create-plugin/templates/common/.config/webpack/webpack.config.ts

```diff
@@ -110,7 +110,7 @@ const config = async (env): Promise<Configuration> => {
             // Keep publicPath relative for host.com/grafana/ deployments
             publicPath: `public/plugins/${pluginJson.id}/img/`,
             outputPath: 'img/',
-            filename: Boolean(env.production) ? '[hash][ext]' : '[name][ext]',
+            filename: Boolean(env.production) ? '[hash][ext]' : '[file]',
```

Comment:
I'm not sure how intuitive it is to keep the folder structure in dev and not in prod. With the current suggestion will the same image file in multiple directories cause webpack to error when doing a prod build?
Would something like this work for both build envs?

- development: `[file]`
- production: `[path][hash][ext]`

Reply:
> will the same image file in multiple directories cause webpack to error when doing a prod build?

As long as images are not identical, prod build is fine because the hash is different

> Would something like this work for both build envs?
>
> development: [file]
> production: [path][hash][ext]

Sure, but I don't think it matters much as long as output files have unique names. Could also change dev to `[name][hash][ext]` to fix the issue. Which do you prefer?

Reply:
Let's leave it as you have it now and get the fix shipped. 👍